airflow/jobs/triggerer_job_runner.py

```diff
@@ -441,6 +441,7 @@ class TriggerDetails(TypedDict):
     task: asyncio.Task
     name: str
     events: int
+    termination_reason: TriggerTerminationReason | None
```

Comment:
This key should be initialized in `create_triggers`

Reply:
Thanks, add the initializing in create_triggers